packages/shopify-codemod/transforms/global-reference-to-import.js

```diff
@@ -131,7 +152,7 @@ export default function globalReferenceToImport(
           body,
           j.importDeclaration([
             j.importDefaultSpecifier(j.identifier(name)),
-          ], j.literal(file))
+          ], j.literal(relativePath(file)))
```

Comment:
Okay, I was reading the diff the wrong way round 😥 This is really fixing some imports where the snake case version was sneaking in.  That's good!

Before/after this transform:
```diff
-import ApplePayCertificatePoller from 'admin/modules/apple_pay_certificate_poller';
+import ApplePayCertificatePoller from 'admin/modules/apple-pay-certificate-poller';
```